products/amazon-rds-postgresql.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Amazon RDS for PostgreSQL
+category: service
+iconSlug: amazonaws
+permalink: /amazon-rds-postgresql
+releasePolicyLink: https://docs.aws.amazon.com/AmazonRDS/latest/PostgreSQLReleaseNotes/postgresql-release-calendar.html
+releaseDateColumn: true
+
+# Auto updates are possible using a custom script.
+# Releases and their date are documented on https://docs.aws.amazon.com/AmazonRDS/latest/PostgreSQLReleaseNotes/postgresql-release-calendar.html
+
+releases:
+-   releaseCycle: "15"
+    releaseDate: 2023-02-28
+    eol: 2027-11-01
+    latest: "15.2"
+    latestReleaseDate: 2023-02-28
+
+-   releaseCycle: "14"
+    releaseDate: 2022-02-03
+    eol: 2026-11-01
+    latest: "14.7"
+    latestReleaseDate: 2023-03-07
+
+-   releaseCycle: "13"
+    releaseDate: 2021-02-24
+    eol: 2025-11-01
+    latest: "13.10"
+    latestReleaseDate: 2023-03-07
+
+-   releaseCycle: "12"
+    releaseDate: 2020-03-31
+    eol: 2024-11-01
+    latest: "12.14"
+    latestReleaseDate: 2023-03-07
+
+-   releaseCycle: "11"
+    releaseDate: 2019-03-13
+    eol: 2023-11-01
+    latest: "11.19"
+    latestReleaseDate: 2023-03-07
+
+-   releaseCycle: "10"
+    releaseDate: 2018-02-27
+    eol: 2023-04-01
+    latest: "10.23"
+    latestReleaseDate: 2023-01-24
+
+-   releaseCycle: "9.6"
+    releaseDate: 2016-11-11
+    eol: 2022-04-30
```

Comment:
April 26th seems to be when this was force-upgraded https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_PostgreSQL.html#PostgreSQL.Concepts.General.DBVersions.Deprecation96

30th should be fine as well, but commenting in case we need it.

Reply:
Ok, thanks. Just to enrich the discussion, the 30th is documented on https://docs.aws.amazon.com/AmazonRDS/latest/PostgreSQLReleaseNotes/postgresql-release-calendar.html as the RDS end of standard support date.